packages/node_modules/@webex/redux-module-spaces/src/__snapshots__/actions.test.js.snap

```diff
@@ -506,7 +506,7 @@ Array [
     "payload": Object {
       "error": Object {
         "code": "Error",
-        "displaySubtitle": "Unable to load spaces. Please try again later.",
+        "displaySubtitle": "Unable to load spaces. Please try again later. [Error]",
```

Comment:
What's the value of adding `[Error]` at the end of the message?

Reply:
This allows users to better identify the cause of a failed load. The only presentation portions of the redux store object that are presented to the user are `displayTitle` and `displaySubtitle`, which aren't dynamic or helpful unless they present some form of error details.

From the JIRA's scope:
>Utilize the redux error store to add an error that will be displayed by the root widget.

Reply:
@lalli-flores  It won't actually display "Error", it will display the error that is being passed to it. Since these are snapshots, it is just what we are passing. Perhaps the test could be updated a bit to have a more realistic error @InteractiveTimmy?

Reply:
These were auto-updated via jest, I can manually improve them though. Let me know if you'd like me to proceed with doing this.

Reply:
OMG! My bad, totally didn't realize this where snapshots 😂🤦🏻‍♀️